digits/scheduler.py

```diff
@@ -214,6 +214,34 @@ def get_job(self, job_id):
             return None
         return self.jobs.get(job_id, None)
 
+    def get_related_jobs(self, job):
+        """
+        Look through self.jobs to try to find the Jobs
+        whose parent contains job
+        """
+        related_jobs = []
+
+        if isinstance(job, ModelJob):
+            datajob = job.dataset
```

Comment:
Thanks for the updates! If you don't mind a couple more comments... there you can do:
```
related_jobs.append(datajob)
```
so you can take lines 233-235 out.